README.md

```diff
@@ -306,6 +327,12 @@ Your help and patience is truly appreciated.
 
 ## Contributing
 
+## Go Compatibility
+
+This project follows the [support policy](https://golang.org/doc/devel/release.html#policy) of Go as its support policy. The two latest major releases of Go are supported by the project.
+
+Currently, that means Go **1.21** or later must be used when including this project as a dependency.
+
```

Comment:
I wasn't sure whether exposing `schemamd` was intentional, so I followed that convention when I added `functionmd`. I can refactor unexport those packages since all they do is create markdown from terraform-json.

Reply:
Ah, fair enough. I'd be fine with just leaving the packages as they are in this PR then coming back **later**:tm: and deprecating/moving internal 😆 

No need to worry about it in this PR 👍🏻